### ISSUE TITLE
Bug: Android LatLngBounds. Random moving

### ISSUE THREAD

**Issue by psinetron** [mapbox/mapbox-gl-native#12320]
When move camera with latLngBounds we have random result for camera zoom.

### Steps to reproduce

 1. move to latlngbounds with radius 500 meter
 2. move latlngBounds with radius 20 km. See zoom
Again
 3. move latlngBounds with radius 10 km. 
4.  move latlngBounds with radius 20 km. See zoom

### Expected behavior
same zoom level

### Actual behavior
random zoom level

### Configuration

**Android versions:** all
**Device models:** xiaomi redmi note, emulators
**Mapbox SDK versions:**  6.2.1

**Comment by tobrun** [comment]
@psinetron thank you for reporting this issue. Due to us using mercator projection, and you defnining a bounds in meters, the generated camera position won't always match. I did a test with solely using LatLng objects and those result in the same camera position every time. 

```
Camera is Target: LatLng [latitude=50.934053267633, longitude=6.97170450000624, altitude=0.0], Zoom:11.938956762782597, Bearing:0.0, Tilt:0.0
Camera is Target: LatLng [latitude=51.960593804726216, longitude=7.6306175000056555, altitude=0.0], Zoom:9.879149440648265, Bearing:0.0, Tilt:0.0
Camera is Target: LatLng [latitude=50.934053267628926, longitude=6.971704500004876, altitude=0.0], Zoom:11.938956762782912, Bearing:0.0, Tilt:0.0
Camera is Target: LatLng [latitude=51.9605938047366, longitude=7.630617499991445, altitude=0.0], Zoom:9.879149440648169, Bearing:0.0, Tilt:0.0
```
- could you verify that your bounds with radius 20 is every time the same?
- are you transforming bearing or tilt in anyway? we recently fixed integration of those with https://github.com/mapbox/mapbox-gl-native/pull/12290.

Closing this issue as not reproducible/actionable, feel free to follow up

**Comment by psinetron** [comment]
@tobrun 
Yes my radius really 20 km at every time.
Yes U transforming bearing and tilt

**Comment by tobrun** [comment]
@psinetron, in that case #12290 will fix your issue, testable with v6.3.0-alpha.2, beta happening this week and final week after.
